wherehows-web/app/templates/components/browser/browser-summary.hbs

```diff
@@ -0,0 +1,29 @@
+<nav role="navigation">
+  <ul class="nav navbar-nav browse-nav">
+    {{#each-in metadata as |entity meta|}}
+      {{#nav-link
+        "browse.entity" entity
+        (query-params page=1 platform="" prefix="")
+        tagName="li"
+        class="col-md-4 browse-nav__entity"}}
+
+        {{#if (and datasetTask.isRunning (eq entity "datasets"))}}
+          {{pendulum-ellipsis-animation}}
+        {{else}}
+
+          <span class="browse-nav__alt">
```

Comment:
Why a span and not a p?

Reply:
👍  got left behind, design change, will revert